symfony/plugins/orangehrmAttendancePlugin/test/fixtures/testcases/AttendanceRecordAPITestCases.yaml

```diff
@@ -962,152 +971,7 @@ Update:
         employeeId: 'E002'
         terminationId: null
 
-  'Attendance Record Update - supervisor to subordinate - Config Enabled - with punchInOffset':
-    userId: 3
-    services:
-      attendance.attendance_service: \OrangeHRM\Attendance\Service\AttendanceService
-      core.config_service: \OrangeHRM\Core\Service\ConfigService
-      admin.user_service: \OrangeHRM\Admin\Service\UserService
-      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
-    factories:
-      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
-    hooks:
-      preAssertValidTestCase: [ '\OrangeHRM\Tests\Attendance\Api\AttendanceRecordAPITest', 'enableSupervisorCanModifyAttendancePreHook' ]
-    attributes:
-      id: 2
-    body:
-      punchInDate: '2022-02-09'
-      punchInTime: '15:30'
-      punchInOffset: '6.5'
-      punchInNote: 'Punched In'
-      punchOutDate: '2022-02-09'
-      punchOutTime: '18:30'
-      punchOutNote: 'Punched Out'
-    data:
-      id: 2
-      punchIn:
-        utcDate: '2022-02-09'
-        utcTime: '09:00'
-        userDate: '2022-02-09'
-        userTime: '15:30'
-        timezoneOffset: '6.5'
-        note: 'Punched In'
-      punchOut:
-        utcDate: '2022-02-09'
-        utcTime: '13:00'
-        userDate: '2022-02-09'
-        userTime: '18:30'
-        timezoneOffset: '5.5'
-        note: 'Punched Out'
-      state:
-        id: 'PUNCHED OUT'
-        name: 'Punched Out'
-      employee:
-        empNumber: 2
-        lastName: 'Abel'
-        firstName: 'Ashley'
-        middleName: 'ST'
-        employeeId: 'E002'
-        terminationId: null
-
-  'Attendance Record Update - supervisor to subordinate - Config Enabled - with punchOutOffset':
```

Comment:
why removed these test cases? :thinking:

Reply:
Previously the requirement was to change the timezone of the employee via passing the timezone offset. But, now whenever we are doing something related to timezone, we need to input both timezone name and timezone offset. New test cases added to cover this scenario and these outdated test cases removed